gulpfile.js

```diff
@@ -204,6 +204,30 @@ function compile(watch, shouldMinify, opt_preventRemoveAndMakeDir,
     wrapper: '<%= contents %>'
   });
 
+  // Entry point for inabox runtime.
+  compileJs('./src/inabox/', 'amp-inabox-babel.js', './dist', {
```

Comment:
Please make sure to file change requests for Cache for canary deployment and `AMP_CONFIG`